### ISSUE TITLE
Bug: Cloud doc should not have version

### ISSUE THREAD

**Issue by xxchan** [risingwavelabs/risingwave-docs-legacy#740]
### Where's the bug?

https://www.risingwave.dev/cloud/intro/

### Describe the bug

Cloud doc should not have version, which can be confusing. Clicking it will go to RW doc. 

BTW, should we mention which RW version is used in RW cloud? Or should we bundle RW cloud doc with the corresponding RW version's doc?

<img width="1341" alt="image" src="https://user-images.githubusercontent.com/37948597/231517479-e3103c39-109e-4ceb-945e-6fb066127769.png">

**Comment by hengm3467** [comment]
@xxchan I don't see the versions dropdown list on my side (could be fixed already):

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/100685635/231614770-60297fce-cbe0-484d-97d7-eccb18ff8bb9.png">

Cloud as a front-end should have only one version, but people might stay on an older version of kernel, given that the automatic and non-breaking kernel upgrade isn't ready yet. We'll have instructions for people to check out their kernel version, and refer to the correct version of kernel docs.

**Comment by CharlieSYH** [comment]
@xxchan Problem solved in https://github.com/risingwavelabs/risingwave-docs/pull/741.
I've added a doc version selection tip in the [Develop with RisingWave Cloud](https://www.risingwave.dev/cloud/develop-overview/) section to show users how to select the correct doc version. That way, we don't have to bundle a new version every time it gets an update. Besides, a cloud user can have multiple clusters which do not necessarily have the same kernel version. For example, I can simultaneously have cluster1 with v0.1.17 (which I created last month) and cluster2 with v0.18.0 (which I created recently).
 
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/100549427/231614471-65ecd2bd-2c8f-457e-9e17-21ed912da562.png">
